### DISCUSSION TITLE
How to change font family or size of DatePicker

### DISCUSSION THREAD

**Discussion by jibin362** [henninghall/react-native-date-picker#297] [Q&A]
The default font used in android is the default one used by android system. Can i use a different phone using styling or some sort like we use custom fonts for Text using fontFamily.

The default font i use in my app is Montesserat which you can see in calendar font but the datepicker is using default phone font.

My Code: 

```
<DatePicker
    date={scrollDate}
    mode="time"
    minuteInterval={15}
    androidVariant="nativeAndroid"
    textColor="#964570"
    onDateChange={(dt) => {
      console.log('date', dt);
      setScrollDate(dt);
    }}
  />
```

Also the active date color initially loads the textColor i give but on scroll changes to white.

On Load

![Screenshot_2021-02-23-14-11-52-98_03268e737c5949f27c8cff3ce3ae24e1](https://user-images.githubusercontent.com/69414163/108820175-cf3e7380-75e1-11eb-861e-4011c3b475e4.jpg)

After Scroll

![Screenshot_2021-02-23-14-11-58-24_03268e737c5949f27c8cff3ce3ae24e1](https://user-images.githubusercontent.com/69414163/108820111-b766ef80-75e1-11eb-96e8-d9619597b378.jpg)

**Comment by tinydiv333** [comment]
Did you get any solution ?

**Reply by jibin362** [reply]
@tinydiv333 We had to change the implementation, since somehow this package didn't met our requirement. So I don't have a proper answer for this question.

We went with a custom implementation with the help of https://github.com/amrulhaq/react-native-wheely-simple-picker.

**Comment by marcinczepielahumanit** [comment]
Hi, recently I've started using this library in a project. 
I can see that there is no update on this topic, but let's try 🙃

Is it possible to add a font-family prop available from the JS side?
Or pass some render wheel picker item function?

We are using Expo with a dev client, so if there is no way to add new props we will be forced to build our custom own solution
